src/main/resources/application.properties

```diff
@@ -1,25 +1,29 @@
 # Zeebe endpoint
 ######################
 # in cloud:
-#zeebe.client.cloud.region=bru-2
-#zeebe.client.cloud.clusterId=XXX
-#zeebe.client.cloud.clientId=XXX
-#zeebe.client.cloud.clientSecret=XXX
+#
+#zeebe.client.cloud.region=syd-1
+#zeebe.client.cloud.clusterId=xxx
+#zeebe.client.cloud.clientId=xx
+#zeebe.client.cloud.clientSecret=xx
+##
 
 # or local
-#zeebe.client.broker.gateway-address=127.0.0.1:26500
-#zeebe.client.security.plaintext=true
+zeebe.client.broker.gateway-address=127.0.0.1:26500
+zeebe.client.security.plaintext=true
 
-
-server.port=8088
+server.port=8080
 management.endpoints.web.exposure.include=*
 
 # Main benchmark configurations
 ######################
-benchmark.startProcesses=true
-benchmark.startPiPerSecond=25
+benchmark.startDecisions=true
+benchmark.startProcesses=false
+benchmark.startDiPerSecond=4500
+benchmark.startPiPerSecond=50
 benchmark.taskCompletionDelay=200
 benchmark.bpmnProcessId=benchmark
+benchmark.dmnDecisionId=pincodedmn
```

Comment:
let's also call this decision something like `decision-table-2x10` that and have working example in the repo